js/ui/camera.js

```diff
@@ -302,7 +303,8 @@ class Camera extends Evented {
      * @param {Object} [options]
      * @param {boolean} [options.linear=false] If `true`, the map transitions using
      *     {@link Map#easeTo}. If `false`, the map transitions using {@link Map#flyTo}. See
-     *     {@link Map#flyTo} for information about the options specific to that animated transition.
+     *     those functions and @{link AnimationOptions} for information about options available.
+     * @param {Function} [options.easing] An easing function for the animated transition. See [AnimationOptions](#AnimationOptions).
      * @param {Function} [options.easing] An easing function for the animated transition.
```

Comment:
> in favor of setting the type of `[options]` to `{AnimationOptions}`?

Good idea. If we go this route, we may also want to:

 - rename/alias `options.linear` to `options.animate`
 - add `options.duration` (this will just be passed to `Map#flyTo`)